tests/conftest.py

```diff
@@ -1920,14 +1920,14 @@ def core_dump_and_config_check(duthosts, request):
             EXCLUDE_CONFIG_TABLE_NAMES = set([])
             # The keys that we don't care
             # Current skipped keys:
-            # 1. "MUX_LINKMGR" table is edited by the `run_icmp_responder` fixture
+            # 1. "MUX_LINKMGR" table is edited by the `run_icmp_responder_session` fixture in dualtor-mixed
             # to account for the lower performance of the ICMP responder/mux simulator,
             # compared to real servers and mux cables. It's appropriate to persist this change
-            # since the testbed will always be using the ICMP responder and mux simulator.
-            # Linkmgrd is the only service to consume this table so it should not affect other test cases.
-            EXCLUDE_CONFIG_KEY_NAMES = [
-                'MUX_LINKMGR|LINK_PROBER'
-            ]
+            # since the testbed will always be using the ICMP responder and mux simulator in dualtor-mixed.
+            if "mixed" in tbinfo["topo"]["name"]:
```

Comment:
Why skip checking `MUX_LINKMGR|LINK_PROBER` for `mixed` topology?

Reply:
As discussed, dualtor-mixed will always run icmp responder, so currently we need to keep linkmgrd prober interval be changed.